learningresources/tests/base.py

```diff
@@ -35,6 +35,13 @@ def logout(self):
         """just a client logout"""
         self.client.logout()
 
+    def assert_status_code(self, url, code, return_body=False):
+        """Asserts the status code of GET attempt"""
+        resp = self.client.get(url, follow=True)
+        self.assertTrue(resp.status_code == code)
+        if return_body:
```

Comment:
I did not want to do it because the function is called `assert_status_code ` and the body is not about that.
In this way it makes more clear when I want the body or not